README.md

```diff
@@ -15,13 +15,20 @@ NOTE: This is also maintained at openshift/aos-cd-jobs repo [https://github.com/
 - scale-ci-linter
 
 ### Properties files
-The parameters/configuration of each Job in the scale-ci-pipeline is supplied through the properties files. It contains key=value pairs, the sample properties for all the supported jobs are hosted in scale-ci/properties.
+The parameters/configuration of each Job in the scale-ci-pipeline is supplied through the properties files. It contains key=value pairs, the sample properties for all the supported jobs are hosted in scale-ci-pipeline/properties-files.
```

Comment:
This is a suggestion to make the sentence a bit more clear:

Potentially change this:

> The parameters/configuration of each Job in the scale-ci-pipeline is supplied through the properties files. It contains key=value pairs, the sample properties for all the supported jobs are hosted in scale-ci-pipeline/properties-files.

to this

> The parameters for each job in the scale-ci-pipeline are supplied through a job specific properties file. It contains key=value pairs, sample properties for all the supported jobs are hosted in scale-ci-pipeline/properties-files.